woocommerce/woocommerce-ios
# Updated WordPressAuthenticator to use Google SignIn 5

Ref. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/281

## Description
We continue to receive a warning from Apple about the deprecated API usage for `UIWebView`. It turns out that the problem lies in the old version of Google SignIn (v.4) which still uses the `UIWebView`. This PR updates the WordPressAuthenticator, that now uses the latest release of Google SignIn.

## Testing
1) Check out this branch
2) run `rake dependencies`
3) Build and run
4) Try to login using Google, and make sure that everything is ok.

## Do not merge until
- The Authenticator PR has been merged. Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/281
- A new Authenticator release has been created
- The release is published on Cocoapods trunk
- This PR points to the new release

Update release notes:
